Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -84,7 +84,7 @@ ProjectData::~ProjectData()
 {
 	delete _geoObjects;
 
-	for(ProcessLib::Process* p : _processes)
+	for(ProcessLib::Process<GlobalSetupType>* p : _processes)
```

Comment:
now it might be good to have a typedef for it. here also auto would suffice, but typedef would also work below.